modules/apps/knowledge-base/knowledge-base-api/src/main/java/com/liferay/knowledge/base/configuration/KBGroupServiceConfiguration.java

```diff
@@ -181,17 +181,18 @@ public interface KBGroupServiceConfiguration {
 	@Meta.AD(deflt = "true", name = "enable-rss", required = false)
 	public boolean enableRSS();
 
-	@Meta.AD(deflt = "20", name = "rss-delta", required = false)
-	public int rssDelta();
+	@Meta.AD(
+		deflt = "${server-property://com.liferay.portal/search.container.page.default.delta}",
+		name = "rss-delta", required = false
+	)
+	public String rssDelta();
```

Comment:
@adolfopa 
My spider-sense is telling me that this can be troublesome...

Can you check it

Reply:
@marcogalluzzi Why did you change the type to `String`? All usages seem to immediately convert it to an integer.

Reply:
I'm using a `String` for `rssDelta` parameter because it's now using the default value from a property file. Using integer now would fail. See similar class _WikiGroupServiceConfiguration_

Reply:
Then, you may need to do an upgrade process. Non string values are stores in a particular format in the configuration. You will need to do some quick test on master, saving the configuration for wiki, and looking into the Configuration_ table.

Reply:
Upgrade process created